### DISCUSSION TITLE
Sony xz2c installation

### DISCUSSION THREAD

**Discussion by linuxitomio** [Divested-Mobile/DivestOS-Build#43] [General]
When trying to install divestos I get this 

    fastboot update ‘/home/noax/Downloads/divested-18.1-20220116-dos-xz2c-fastboot.zip’
    extracting android-info.txt (0 MB)…
    extracting boot.img (64 MB)…
    target reported max download size of 805306368 bytes
    archive does not contain ‘boot.sig’
    archive does not contain ‘boot_other.img’
    extracting dtbo.img (8 MB)…
    archive does not contain ‘dtbo.sig’
    archive does not contain ‘dt.img’
    archive does not contain ‘recovery.img’
    extracting system.img (1575 MB)…
    archive does not contain ‘system.sig’
    archive does not contain ‘system_other.img’
    extracting vbmeta.img (0 MB)…
    archive does not contain ‘vbmeta.sig’
    extracting vendor.img (604 MB)…
    archive does not contain ‘vendor.sig’
    archive does not contain ‘vendor_other.img’
    Bootloader Version…: 1310-7079_X_Boot_SDM845_LA2.0.1_Q_207
    Baseband Version…: 1311-5320_52.1.A.3.49
    Serial Number…: BH902BX0BZ

    checking product…
    FAILED

    Device product is ‘H8324’.
    Update requires ‘sdm845’.

    finished. total time: 0.008s
    noax@noax:~$ fastboot erase avb_custom_key
    erasing ‘avb_custom_key’…
    FAILED (remote: No such partition.)
    finished. total time: 0.001s
    noax@noax:~$ fastboot flash avb_custom_key ‘/home/noax/Downloads/avb_pkmd.bin’
    target reported max download size of 805306368 bytes
    sending ‘avb_custom_key’ (1 KB)…
    OKAY [ 0.005s]
    writing ‘avb_custom_key’…
    FAILED (remote: No such partition.)
    finished. total time: 0.006s

**Comment by SkewedZeppelin** [comment]
Please try flashing divested-18.1-20220116-dos-xz2c.zip via TWRP or LineageOS
recovery then.

Otherwise perhaps update your fastboot binary.

**Reply by linuxitomio** [reply]
the twrp for this device doesnt exist anymore in their web.

And what you mean with update fastboot? Isnt that what I have done in the first line of the commands?
fastboot update ‘/home/noax/Downloads/divested-18.1-20220116-dos-xz2c-fastboot.zip’

I have just try  it with lineage recovery and first it says that the signature verification failed and  after clicking install anyway it prompts that this will  downgrade your phone cause the timestamp it should be newer than 1642736834 cause this build is 1642736834.
even though, the system seems to be install in the device.

**Comment by linuxitomio** [comment]
What do you need from this device to change the status from untested to tested? And is it gonna be change the installation method in it?

And I might say that UnifiedNIp app and divestos settings says that is build for an old system build and I should update? 

Do you want me to check this device or not? Can you help me with the fastboot and the avd to try to lock the bootloader?

What steps to do to check this device?
